jsapp/js/components/modalForms/encryptForm.es6

```diff
@@ -152,7 +152,7 @@ class EncryptForm extends React.Component {
 
             <Button
               type='bare'
-              color='storm'
+              color='dark-blue'
```

Comment:
didn't check this one; encryption modal is not currently enabled ([xref](https://github.com/kobotoolbox/kpi/issues/2692#ref-pullrequest-753980535))

Reply:
I had to change few lines of code to make it appear, so (for what it's worth) I was able to test it :)